### PR TITLE
zh-update: /faq/applications/redis.md

### PR DESCRIPTION
**Pull request by yuxiaobo96**
ref:https://github.com/servicemesher/istio-official-translation/issues/1737

### CODE REVIEW

File: content/zh/faq/applications/redis.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+---
+title: 我可以在 Istio 网格内运行 Redis 吗？
+description: 如何使用 Istio 运行 Redis。
+weight: 50
+keywords: [redis]
+---
+
+与在Istio服务网格中部署的其他服务类似，Redis 实例需要监听 `localhost` (`127.0.0.1`)。但是，每个 Redis 从属实例都应声明一个地址，主服务器可以使用该地址来访问它，该地址不能是 `localhost` (`127.0.0.1`)。
```
Comment (gorda) [inline content/zh/faq/applications/redis.md]:
```suggestion
与在 Istio 服务网格中部署的其他服务类似，Redis 实例需要监听 `localhost` (`127.0.0.1`)。每个 Redis 从属实例都应声明一个地址，主服务器可以使用该地址来访问它，但是，该地址不能是 `localhost` (`127.0.0.1`)。
```

File: content/zh/faq/applications/redis.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+使用 Redis 配置参数 `replica-announce-ip` 来公布正确的地址。例如，使用以下步骤将 `replica-announce-ip` 设置为每个 Redis 从属实例的 IP 地址：
+
+通过 `env` 小节中的环境变量传递 Pod IP 地址从属 `StatefulSet` 的定义：
```
Comment (gorda) [inline content/zh/faq/applications/redis.md]:
```suggestion
通过从属 `StatefulSet` 的 `env` 小节中定义的环境变量传递 Pod IP 地址：
```

File: content/zh/faq/applications/redis.md (hunk 1 above)
Comment (yuxiaobo96) [reply content/zh/faq/applications/redis.md]: done

File: content/zh/faq/applications/redis.md (hunk 2 above)
Comment (yuxiaobo96) [reply content/zh/faq/applications/redis.md]: Thanks for your advice.
done

**Pull request merged**